<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopicWidget</class>
 <widget class="QWidget" name="TopicWidget">
  <layout class="QVBoxLayout" name="mainLayout">

   <!-- Header -->
   <item>
    <layout class="QHBoxLayout" name="headerLayout">

     <item>
      <widget class="QLabel" name="labelTopic">
       <property name="text">
        <string>Тема:</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLineEdit" name="name_edit">
       <property name="placeholderText">
        <string>Название темы</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="add_activity_btn">
       <property name="text">
        <string>+ Добавить занятие</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="delete_btn">
       <property name="text">
        <string>Удалить тему</string>
       </property>
      </widget>
     </item>

     <item>
      <spacer name="headerSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>

    </layout>
   </item>

   <!-- Summary -->
   <item>
    <layout class="QHBoxLayout" name="summaryLayout">

     <item>
      <widget class="QLabel" name="lecture_label">
       <property name="text">
        <string>Лекции: 0</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLabel" name="practice_label">
       <property name="text">
        <string>Практики: 0</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLabel" name="self_study_label">
       <property name="text">
        <string>Сам.раб.: 0</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLabel" name="sdo_label">
       <property name="text">
        <string>СДО: 0</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLabel" name="total_label">
       <property name="text">
        <string>Всего: 0</string>
       </property>
      </widget>
     </item>

     <item>
      <spacer name="summarySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>

    </layout>
   </item>

   <!-- Activities container -->
   <item>
    <layout class="QVBoxLayout" name="activities_container"/>
   </item>

  </layout>
 </widget>

 <resources/>
 <connections/>
</ui>
